docs/using_pyodide_from_javascript.md

```diff
@@ -102,6 +101,101 @@ pyodide.loadPackage('matplotlib').then(() => {
 });
 ```
 
+## Alternative way to load packages and run Python code
+
+Alternatively you can run Python code without manually pre-loading packages. You can do this with [pyodide.runPythonAsync](api_reference.md#pyodideloadpackagenames-messagecallback-errorcallback) function, which will automatically download all packages that the code snippet imports.
+
+Note: although the function is called Async, it still blocks the main thread. To run Python code asynchronously see [WebWorker](using_pyodide_from_webworker.md)
+
+## Alternative Example
+
+```html
+<!DOCTYPE html>
+<head>
+    <script type="text/javascript">
+        window.languagePluginUrl = 'https://pyodide-cdn2.iodide.io/v0.15.0/full/';
+    </script>
+    <script src="https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js"></script>
+</head>
+
+<body>
+  <p>You can execute any Python code. Just enter something in the box below and click the button.</p>
+  <input id='code' value='sum([1,2,3,4,5])'>
```

Comment:
Maybe

```suggestion
  <input id='code' value='import numpy as np; np.sum([1,2,3,4,5])'>
```
otherwise this example wouldn't really illustrate how auto-load packages with runPythonAsync